webapp/components/loading.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react'
 
-const Loading = () => (
-  <div className="notfound" style={{ minHeight: '90vh' }}>
-    <img src="/img/tucan.svg" alt="tucan" />
-    <p className="subhead">
-      <strong>Loading... </strong>
-    </p>
-  </div>
-)
+import * as PanEuropean from '@common/assessment/panEuropean'
+
+const Loading = () => {
+  // Toucan is hidden in PanEuropean pages.
+  const { pathname } = window.location
```

Comment:
Should we use
`import { useLocation } from 'react-router-dom'`?

Reply:
it's the same